src/eopf_geozarr/s2_optimization/s2_multiscale.py

```diff
@@ -3,770 +3,824 @@
+        zoom_for_height = max(0, int(np.ceil(np.log2(height / tile_width))))
+        zoom = max(zoom_for_width, zoom_for_height)
+
+        # Calculate scale factor relative to finest resolution
+        finest_res_meters = res_order[all_resolutions[0]]
+        scale_factor = res_meters // finest_res_meters
+
+        # calculate translation relative to finest resolution
+        trans_meters = (res_meters - finest_res_meters) / 2
+
+        overview_levels.append(
+            {
+                "level": res_name,  # Use string-based level name
+                "zoom": zoom,
+                "width": width,
+                "height": height,
+                "translation_relative": trans_meters,
+                "scale_absolute": res_meters,
+                "scale_relative": scale_factor,
+                "chunks": dataset.data_vars[first_var.name].chunks,
+            }
         )
-        if native_bounds is None:
-            if verbose:
-                log.info(
-                    "No bounds found, skipping multiscales metadata",
-                    base_path=base_path,
-                )
-            return
-
-        # Create overview_levels structure with string-based level names
-        overview_levels = []
-        for res_name in all_resolutions:
-            res_meters = res_order[res_name]
-
-            dataset = res_groups[res_name]
-
-            if dataset is None:
-                continue
-
-            # Get first data variable to extract dimensions
-            first_var = next(iter(dataset.data_vars.values()))
-            height, width = first_var.shape[-2:]
-
-            # Calculate zoom level (higher resolution = higher zoom)
-            tile_width = 256
-            zoom_for_width = max(0, int(np.ceil(np.log2(width / tile_width))))
-            zoom_for_height = max(0, int(np.ceil(np.log2(height / tile_width))))
-            zoom = max(zoom_for_width, zoom_for_height)
 
-            # Calculate scale factor relative to finest resolution
-            finest_res_meters = res_order[all_resolutions[0]]
-            scale_factor = res_meters // finest_res_meters
+    if len(overview_levels) < 2:
+        log.info("    Could not create overview levels for {}", base_path=base_path)
+        return
 
-            overview_levels.append(
-                {
-                    "level": res_name,  # Use string-based level name
-                    "zoom": zoom,
-                    "width": width,
-                    "height": height,
-                    "scale_factor": scale_factor,
-                    "chunks": dataset.data_vars[first_var.name].chunks,
-                }
-            )
-
-        if len(overview_levels) < 2:
-            if verbose:
-                log.info(
-                    "    Could not create overview levels for {}", base_path=base_path
-                )
-            return
+    multiscales: TMSMultiscalesAttrsJSON | MultiscalesAttrsJSON
 
+    if multiscales_flavor == "ogc_tms":
```

Comment:
Dont you think we could simply have both multiscales flavor combined until we are aligned and https://github.com/EOPF-Explorer/data-model/issues/77 is implemented and integrated in titiler?